profile/README.md

```diff
@@ -0,0 +1,12 @@
+Valkey is an open source, in memory datastore released under the [BSD-3 Clause License](https://github.com/valkey-io/valkey?tab=BSD-3-Clause-1-ov-file#readme). It is a continuation of the work that was being done on Redis 7.2.4. 
+
+## Community 🎉
+
+- 🎙️ Chat with us on [Slack](https://join.slack.com/t/valkey-oss-developer/shared_invite/zt-2haj75jfq-3xj~tErWhedsldu~4UxNPw) or [Discord](https://discord.gg/zbcPa5umUB) 
```

Comment:
The slack invites expire after 30 days, we haven't decided if we want to open up slack to everyone yet. We do encourage matrix though https://github.com/valkey-io/valkey/blob/unstable/.github/ISSUE_TEMPLATE/config.yml#L10

Reply:
Or maybe just link here? https://valkey.io/connect.html

Reply:
I like the connect page the best. Resolved in my most recent commit. That way there is only one place to update links. 

I do think it would be good to link to slack or discord. Matrix is cool but IMO it's hard for people to get involved and stay involved places like that.